osmaxx/conversion/management/commands/result_harvester.py

```diff
@@ -56,7 +56,7 @@ def _update_job(self, job_id, queue):
         if job is None:  # already processed by someone else
             conversion_job.refresh_from_db()
             if conversion_job.status not in FINAL_STATUSES:
-                logger.warning("job {} not found in queue but status is {} on database.".format(
```

Comment:
As `job_id` can apparently be `None`, should we log the ID of `conversion_job`, too?